rest-service/manager_rest/rest/rest_utils.py

```diff
@@ -703,8 +710,12 @@ def decrease_deployment_counts_in_graph(self, source):
                 )
                 if target.sub_services_count:
                     target.sub_services_count -= total_services
+                    if not target.sub_services_count:
+                        target.sub_services_count = None
```

Comment:
Out of interest, and bearing all the previous logic in mind: Why are we ever allowing these to be null values- why not instead require them to be integers, defaulting to 0- that seems like it would simplify a lot of this code.

Reply:
Yup will make default to zero, already left comment above